### DISCUSSION TITLE
Which countries have more than 5000 museums?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#722] [Galaxy Trivia] [answered]
*answer with country flags*

**Comment by peterbaccaro** [comment]
🇺🇸 - US
🇩🇪 - GERMANY
🇯🇵 - JAPAN
🇷🇺 - RUSSIA

**Comment by SANKARAMDAS** [comment, marked as answer]
🇺🇸 - US
🇩🇪 - GERMANY
🇯🇵 - JAPAN
🇷🇺 - RUSSIA

According to UNESCO, the United States recorded the highest number of museums globally, with approximately 33,082 institutions as of March 2021.

**Comment by thuongtruong1009** [comment]
Here are some countries that have more than 5000 museums:

🇺🇸 United States
🇷🇺 Russia
🇩🇪 Germany
🇫🇷 France
🇮🇹 Italy
🇨🇳 China
🇯🇵 Japan
🇬🇧 United Kingdom
🇪🇸 Spain
🇮🇳 India

Please note that the number of museums may vary, and the list provided is based on general information up until my knowledge cutoff in September 2021.
